### ISSUE TITLE
tox check fails with python3.8

### ISSUE THREAD

**Issue by mxmilkiib** [arsenetar/dupeguru#668]
# Bug / issue Report

Arch has python3.8 not python3.6 or python3.7.

```
System Information:
 - DupeGuru Version: `version`
 - Operating System: `Windows/Linux/OSX` `distribution` `version` 
==> Starting check()...
py36 create: /home/milk/.cache/yay/dupeguru-git/src/dupeguru/.tox/py36
SKIPPED: InterpreterNotFound: python3.6
py37 create: /home/milk/.cache/yay/dupeguru-git/src/dupeguru/.tox/py37
SKIPPED: InterpreterNotFound: python3.7
py38 create: /home/milk/.cache/yay/dupeguru-git/src/dupeguru/.tox/py38
py38 installdeps: -r/home/milk/.cache/yay/dupeguru-git/src/dupeguru/requirements.txt, -r/home/milk/.cache/yay/dupeguru-git/src/dupeguru/requirements-extra.txt
py38 installed: alabaster==0.7.12,appdirs==1.4.4,attrs==19.3.0,Babel==2.8.0,black==19.10b0,certifi==2020.4.5.1,chardet==3.0.4,click==7.1.2,distlib==0.3.0,docutils==0.16,filelock==3.0.12,flake8==3.8.2,hsaudiotag3k==1.1.3,idna==2.9,imagesize==1.2.0,Jinja2==2.11.2,MarkupSafe==1.1.1,mccabe==0.6.1,packaging==20.4,pathspec==0.8.0,pluggy==0.13.1,polib==1.1.0,py==1.8.1,pycodestyle==2.6.0,pyflakes==2.2.0,Pygments==2.6.1,pyparsing==2.4.7,pytest==2.9.2,pytest-monkeyplus==1.1.0,pytz==2020.1,regex==2020.5.14,requests==2.23.0,Send2Trash==1.5.0,six==1.15.0,snowballstemmer==2.0.0,Sphinx==3.0.4,sphinxcontrib-applehelp==1.0.2,sphinxcontrib-devhelp==1.0.2,sphinxcontrib-htmlhelp==1.0.3,sphinxcontrib-jsmath==1.0.1,sphinxcontrib-qthelp==1.0.3,sphinxcontrib-serializinghtml==1.1.4,toml==0.10.1,tox==3.15.1,tox-travis==0.12,typed-ast==1.4.1,urllib3==1.25.9,virtualenv==20.0.21
py38 run-test-pre: PYTHONHASHSEED='1160615430'
py38 run-test: commands[0] | make modules
make: Nothing to be done for 'modules'.
py38 run-test: commands[1] | flake8
./hscommon/build.py:560:20: E741 ambiguous variable name 'l'
ERROR: InvocationError for command /home/milk/.cache/yay/dupeguru-git/src/dupeguru/.tox/py38/bin/flake8 (exited with code 1)
__________________________________________________________________________________________________________________ summary __________________________________________________________________________________________________________________
SKIPPED:  py36: InterpreterNotFound: python3.6
SKIPPED:  py37: InterpreterNotFound: python3.7
ERROR:   py38: commands failed
==> ERROR: A failure occurred in check().
    Aborting...
Error making: dupeguru-git
```

**Comment by arsenetar** [comment]
I have fixed this over at https://github.com/arsenetar/dupeguru/commit/e921b9827e3ea77f925a2289392708d622df505c which is part of a branch targeted at #653 (found it when working on that issue with the tox test).  I needed to finish up a few more things over there then that will be merged in.

**Comment by arsenetar** [comment]
@mxmilkiib this should be fixed in #672
